release/release_tests.yaml

```diff
@@ -2739,6 +2739,42 @@
 # Learning and benchmarking tests
 # ----------------------------------------------------------
 
+# --------------------------
+# DreamerV3
+# --------------------------
+# TODO (sven): Move algo and this test to pytorch
+- name: rllib_learning_tests_pong_dreamerv3_tf2
+  group: RLlib tests
+  working_dir: rllib_tests
+
+  stable: true
+
+  frequency: weekly
+  team: rllib
+  cluster:
+    byod:
+      type: gpu
+      post_build_script: byod_rllib_test.sh
+      runtime_env:
+        - RLLIB_TEST_NO_JAX_IMPORT=1
+        - LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/ray/.mujoco/mujoco210/bin
+    cluster_compute: 1gpu_4cpus.yaml
```

Comment:
this is using `p2.xlarge` which is about [$1/hour](https://instances.vantage.sh/aws/ec2/p2.xlarge); from cost perspective, i think it's absolutely fine to use this node for 12 hours per week

from the weekly release perspective, we are thinking about reduce many 24-hour tests to be 8-hour tests, but no concrete plan yet so should not be a blocker